build/MPack.targets

```diff
@@ -70,10 +69,17 @@
       <AddinInfoLines Include="&lt;/Addin&gt;" />
     </ItemGroup>
 
+    <ItemGroup>
+      <LanguageServiceAssemblies Include="$(LanguageServiceOutputPath)%(LanguageServiceAssemblyNames.Identity)" />
+      <LanguageServicePDBs Include="%(LanguageServiceAssemblies.RootDir)%(Directory)%(FileName).pdb" Condition="Exists('%(LanguageServiceAssemblies.RootDir)%(Directory)%(FileName).pdb')" />
```

Comment:
Generally MSBuild item names aren't plural

Reply:
ooo good to know. I'll update all the names in here.